### PR TITLE
connectors: converters page update

### PR DESCRIPTION
**Pull request by bochenekmartin**

### CODE REVIEW

**Review by tomasz-sadura — APPROVED** [review]

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc [hunk 1]
```diff
@@ -2,13 +2,23 @@
 :description: Use converters to handle the serialization and deserialization of data between a Redpanda topic and a managed connector.
 :page-cloud: true
 
-Converters handle the serialization and deserialization of data between a Redpanda topic and a connector. 
+Connectors are a translation layer working between the Redpanda and the remote system.
```
Comment (Deflaimun) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc]:
```suggestion
Connectors are a translation layer working between Redpanda and the remote system. 
```

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc [hunk 2]
```diff
@@ -2,13 +2,23 @@
 :description: Use converters to handle the serialization and deserialization of data between a Redpanda topic and a managed connector.
 :page-cloud: true
 
-Converters handle the serialization and deserialization of data between a Redpanda topic and a connector. 
+Connectors are a translation layer working between the Redpanda and the remote system. 
+
+For *sink* connectors the translation happens in the following phases:
+
+. Converter deserializes data from Redpanda message format (for example JSON, Avro) to a universal in-memory Connect data format.
```
Comment (Deflaimun) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc]:
```suggestion
. Converter deserializes data from Redpanda message format (for example JSON or Avro) to a universal in-memory connect data format.
```

Comment (Deflaimun) [reply modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc]: either connect lower case, or Redpanda Connect

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc [hunk 3]
```diff
@@ -74,8 +84,7 @@ See also: https://debezium.io/documentation/reference/2.2/integrations/cloudeven
 == Protobuf converter {badge-cloud-beta}
 
 The Protobuf converter requires a schema in Schema Registry.
-Currently the converter only supports sink connectors.
-It works for primitive types (numbers, boolean, string) and enum types.
+Currently the converter supports sink connectors only. Source connectors are not yet supported.
```
Comment (Deflaimun) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc]:
```suggestion
The converter only supports sink connectors. Source connectors are not supported.
```

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc [hunk 4]
```diff
@@ -2,13 +2,23 @@
...
+
+. Connector translates the data model from remote system format to the in-memory Connect data structure.
+. Converter serializes the data from a universal in-memory Connect format to a Redpanda message.
 
 Each Redpanda message is a key and value record. Record key and value converters are configured separately with the `Redpanda message key format` and `Redpanda message value format` properties. Key and value converters can be different.
 
 [NOTE]
 ====
-If an external system requires structured data (like BigQuery or a SQL database), then you must provide data with a schema. Use the Avro, Protobuf, or JSON converter with a schema.
+If an external system requires structured data (like BigQuery or a SQL database), then you must provide data with a schema. Use the Avro, Protobuf or JSON converter with a schema.
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc]: This is the Oxford comma, which we adhere to.
https://github.com/redpanda-data/docs-site/blob/main/meta-docs/STYLE-GUIDE.adoc#commas

Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc]: So please delete this update.

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc (hunk 1 above)
Comment (Feediver1) [reply modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc]: Perhaps:
Connectors are a translation layer working between Redpanda and a remote system.

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc [hunk 5]
```diff
@@ -2,13 +2,23 @@
 :description: Use converters to handle the serialization and deserialization of data between a Redpanda topic and a managed connector.
 :page-cloud: true
 
-Converters handle the serialization and deserialization of data between a Redpanda topic and a connector. 
+Connectors are a translation layer working between the Redpanda and the remote system. 
+
+For *sink* connectors the translation happens in the following phases:
+
+. Converter deserializes data from Redpanda message format (for example JSON or Avro) to a universal in-memory connect data format.
+. The in-memory Connect data structure is translated by the connector to the data model of the remote system.
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc]: I think we only need to capitalize when talking about specific connectors by name.

Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc]:
```suggestion
. The in-memory connect data structure is translated by the connector to the data model of the remote system.
```

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc [hunk 6]
```diff
@@ -2,13 +2,23 @@
...
+
+For *source* connectors it is vice versa, the phases are:
+
+. Connector translates the data model from remote system format to the in-memory Connect data structure.
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc]:
```suggestion
. Connector translates the data model from remote system format to the in-memory connect data structure.
```

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc [hunk 7]
```diff
@@ -2,13 +2,23 @@
...
+. Converter serializes the data from a universal in-memory Connect format to a Redpanda message.
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc]:
```suggestion
. Converter serializes the data from a universal in-memory connect format to a Redpanda message.
```

**Review by Feediver1 — APPROVED** [review]
See the suggestions in review

**Pull request merged**